ember-toucan-core/src/components/form/radio-group-field.hbs

```diff
@@ -0,0 +1,19 @@
+<Form::Fieldset
+  @label={{@label}}
+  @hint={{@hint}}
+  @error={{@error}}
+  @isDisabled={{@isDisabled}}
+  ...attributes
+>
+  {{yield
+    (hash
+      RadioField=(component
+        (ensure-safe-component this.RadioFieldComponent)
+        name=@name
+        onChange=this.handleInput
+        selectedValue=@value
+        isDisabled=@isDisabled
+      )
```

Comment:
Suggestion: Shouldn't we also yield the `selectedValue` in case the user wants to format that in some way?

Reply:
NVM - read the thread above!